tripleo-heat-templates-diff/diff_OSPD13_8.0.7-21

```diff
@@ -0,0 +1,15 @@
+diff -r -u -N openstack-tripleo-heat-templates-original/docker/services/neutron-api.yaml openstack-tripleo-heat-templates/docker/services/neutron-api.yaml
```

Comment:
I would argue that these lines for odl are subject to change. Why not these lines:

https://github.com/openstack/tripleo-heat-templates/blob/stable/queens/docker/services/neutron-api.yaml#L242-L246

Reply:
When we apply this patch the nuage code will be added in between these lines https://github.com/openstack/tripleo-heat-templates/blob/stable/queens/docker/services/neutron-api.yaml#L246-L247 and this is what we tested and suggesting to users as well. We are seeing odl lines in the below diff because, the diff mechanism only takes 3 lines above and below. Let me know if we have to change the location of our nuage code.

Reply:
Moreover, we provide apply patch based on openstack-tripleo-heat-templates version and if there are any changes it will be only present in new (different) version of openstack-tripleo-heat-templates